### PR TITLE
Add Dashboard Access, Overview and Profiling Page

### PR DESCRIPTION
**Pull request by breeswish**
### What is changed, added or deleted? (Required)

Add docs for TiDB Dashboard: Access, Overview and Profiling page.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): #3250

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -360,16 +360,26 @@
...
-      + 访问
-      + 概况页面
-      + 集群信息页面
-      + 流量可视化页面
-      + SQL 语句分析页面
-      + 慢查询页面
-      + 集群诊断页面
-      + 日志搜索页面
-      + 实例性能分析页面
+    + TiDB Dashboard
```
Comment (scsldb) [inline TOC.md:363]: 请将相关的人名去掉

File: dashboard/access.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# 访问 TiDB Dashboard
+
+通过浏览器访问 <http://127.0.0.1:2379/dashboard> （请将 127.0.0.1:2379 替换为实际 PD 实例地址和端口）即可打开 TiDB Dashboard。
+
+> **注意：**
+>
+> 部署了多个 PD 组件时，填写任意一个 PD 组件地址都可以访问 TiDB Dashboard，但其中仅有一个 PD 组件会真正运行 TiDB Dashboard。访问其他 PD 组件时浏览器都将会被重定向到该 PD 组件。
+> 当防火墙配置不正确时，可能出现无法访问 TiDB Dashboard 的情况。
```
Comment (scsldb) [inline dashboard/access.md:8]: 防火墙配置不正确具体是指什么？如果不复杂可以在这里说明一下

File: dashboard/access.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## 登录
+
+首次访问 TiDB Dashboard 将会显示用户登录界面，如下图所示。需使用与访问 TiDB 数据库一致的 SQL 用户 root 的密码登录。
```
Comment (scsldb) [inline dashboard/access.md:12]: 这里直接讲要求 root 账号登录是否就 OK

File: dashboard/access.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+![登录界面](/media/dashboard/access/login.png)
+
+如果存在以下情况，则可能会登录失败：
```
Comment (scsldb) [inline dashboard/access.md:16]: 网络不通？前面讲的防火墙未正确配置呢？

File: dashboard/access.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+登录后，24 小时内将保持自动登录状态。参见「[登出](#登出)」章节了解如何登出用户。
+
+## 切换语言
+
+在登录界面中，可点击 Switch Language 下拉框切换界面显示语言：
+
+![切换语言](/media/dashboard/access/switch-language.png)
+
+目前支持以下语言：
```
Comment (scsldb) [inline dashboard/access.md:31]: 可以提到最前面，先知道有哪些语言再去看如何切换逻辑更顺

File: dashboard/access.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- 简体中文
+- 英文
+
+## 登出
+
+登录后，在左侧导航处点击登录用户名，可切换到用户页面。在用户页面点击「登出」（Logout）按钮即可登出当前用户。登出后，需重新输入用户名密码。
+
+![登出](/media/dashboard/access/logout.png)
+
+## 浏览器兼容性
```
Comment (scsldb) [inline dashboard/access.md:42]: 《浏览器兼容性》放到最前面，使用的时候先要知道这类信息，其实也可放注意事项

File: dashboard/dashboard-overview.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
+---
+title: 概况页面
```
Comment (TomShawn) [inline dashboard/dashboard-overview.md:2]:
```suggestion
title: TiDB Dashboard 概况页面
```

File: dashboard/dashboard-overview.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+category: how-to
+---
+
+# 概况页面
```
Comment (TomShawn) [inline dashboard/dashboard-overview.md:6]:
```suggestion
# TiDB Dashboard 概况页面
```

File: dashboard/dashboard-profiling.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+## 访问
+
+可以通过以下两种方法访问实例性能分析页面：
+
+- 登录后，左侧导航条点击 **高级调试** → **实例性能分析**：
+
+  ![访问](/media/dashboard/profiling-access.png)
+
+- 在浏览器中访问 <http://127.0.0.1:2379/dashboard/#/instance_profiling>（将 127.0.0.1:2379 替换为任意实际 PD 地址和端口）。
```
Comment (HunDunDM) [inline dashboard/dashboard-profiling.md:18]:
```suggestion
- 在浏览器中访问 <http://127.0.0.1:2379/dashboard/#/instance_profiling>（将 `127.0.0.1:2379` 替换为实际 PD 实例地址和端口）。
```

**Comment by breeswish** [comment]
@lilin90 @yikeke PTAL, thanks!

**Comment by TomShawn** [comment]
@breeswish Please change the image file names according to https://github.com/pingcap/docs-cn/pull/3359.

File: dashboard/dashboard-overview.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+>
+> 该功能仅在部署了 Prometheus 监控组件的集群上可用，未部署监控组件的情况下会显示为失败。
+
+## Top SQL 语句
+
+该区域显示最近一段时间内整个群集中累计耗时最长的 10 类 SQL 语句。查询参数不一样但结构一样的 SQL 会归为同一类 SQL 语句，在同一行中显示：
+
+![界面](/media/dashboard/dashboard-overview-top-statements.png)
+
+该区域显示的内容与 [SQL 语句分析页面](/dashboard/dashboard-statements-list.md)一致，可点击 **Top SQL 语句**（Top SQL Statements）标题查看完整列表。关于该表格中各列详情，见 [SQL 语句分析页面](/dashboard/dashboard-statements-list.md)。
```
Comment (TomShawn) [inline dashboard/dashboard-overview.md:49]: 这是个死链

**Pull request merged**